subprojects/docs/src/snippets/java/fixtures/groovy/lib/build.gradle

```diff
@@ -46,7 +46,8 @@ components.java.withVariantsFromConfiguration(configurations.testFixturesRuntime
 // end::disable-test-fixtures-publishing[]
 
 tasks.create("usages") {
+    def javaComponent = components.java.usages.collect { it.name }
```

Comment:
Thanks for this @3flex. Could you please a more accurate name for the local variable (`variantNames`, `usages` etc)? (here and in the kotlin version)

Reply:
Apologies, that was sloppy! I went with `javaComponentUsages`, hope that's appropriate.